pkg/virt-operator/resource/apply/core_test.go

```diff
@@ -62,13 +62,10 @@ var _ = Describe("Apply", func() {
 			service.Spec.Type = corev1.ServiceTypeClusterIP
 			service.Spec.ClusterIP = ""
 
-			ops, err := generateServicePatch(cachedService, service)
-			Expect(err).To(BeNil())
-			Expect(ops).ToNot(Equal(""))
+			Expect(generateServicePatch(cachedService, service)).To(BeEmpty())
 		})
 
-		It("should replace if ClusterIp != \"\" during update and ip changes", func() {
-
+		It("should replace if ClusterIp is empty during update and ip changes", func() {
```

Comment:
/lgtm cancel
I think it should be "non-empty" here, right?

Reply:
fixed